lib/web_ui/lib/src/engine/text/ruler.dart

```diff
@@ -18,33 +18,12 @@ String buildCssFontString({
   required double? fontSize,
   required String fontFamily,
 }) {
-  final StringBuffer result = StringBuffer();
+  final String cssFontStyle = fontStyle?.toCssString() ?? StyleManager.defaultFontStyle;
+  final String cssFontWeight = fontWeight?.toCssString() ?? StyleManager.defaultFontWeight;
+  final num cssFontSize = fontSize?.floor() ?? StyleManager.defaultFontSize;
```

Comment:
It probably doesn't matter much, but:

```suggestion
  final int cssFontSize = fontSize?.floor() ?? StyleManager.defaultFontSize;
```